core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -325,7 +325,8 @@ private[spark] object UIUtils extends Logging {
         <div class="container-fluid">
           <div class="row">
             <div class="col-12">
-              <h3 style="vertical-align: bottom; display: inline-block;">
+              <h3 style="vertical-align: bottom; white-space: nowrap; overflow: hidden;
+               text-overflow: ellipsis;">
```

Comment:
It's better to have one more whitespace.

Reply:
Thanks @sarutak done this!